folding-schemes/src/folding/nova/mod.rs

```diff
@@ -283,15 +283,15 @@ where
 /// Implements Nova+CycleFold's IVC, described in [Nova](https://eprint.iacr.org/2021/370.pdf) and
 /// [CycleFold](https://eprint.iacr.org/2023/1192.pdf), following the FoldingScheme trait
```

Comment:
maybe adding a comment on what's `H`, as in the `CommitmentScheme` comment: [sonobe/folding-schemes/src/commitment/mod.rs#L13](https://github.com/privacy-scaling-explorations/sonobe/blob/3099befc6a7c7d9a3a35b083cacfb0fa393b2cd9/folding-schemes/src/commitment/mod.rs#L13)

Reply:
Good suggestion, added it [here](https://github.com/privacy-scaling-explorations/sonobe/blob/19ca02989ad77d8b598374b74757cdc8a4dd7c52/folding-schemes/src/folding/nova/mod.rs#L289)